gen/scim_atlassian_mu

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use perunServicesInit;
+use perunServicesUtils;
+use open qw/:std :utf8/;
+use JSON::XS;
+use utf8;
+
+local $::SERVICE_NAME = "scim_atlassian_mu";
+local $::PROTOCOL_VERSION = "3.0.0";
+my $SCRIPT_VERSION = "3.0.0";
+
+perunServicesInit::init;
+my $DIRECTORY = perunServicesInit::getDirectory;
+my $data = perunServicesInit::getHashedDataWithGroups;
+
+#forward declaration
+sub processUsers;
+sub processGroups;
+sub processMemberships;
+
+#Constants
+our $A_USER_ID;                   *A_USER_ID =               \'urn:perun:user:attribute-def:core:id';
+our $A_USER_LOGIN;                *A_USER_LOGIN =            \'urn:perun:user_facility:attribute-def:virt:login';
+our $A_USER_GIVEN_NAME;           *A_USER_GIVEN_NAME =       \'urn:perun:user:attribute-def:core:firstName';
+our $A_USER_FAMILY_NAME;          *A_USER_FAMILY_NAME =      \'urn:perun:user:attribute-def:core:lastName';
+our $A_GROUP_NAME;                *A_GROUP_NAME =            \'urn:perun:group:attribute-def:core:name';
```

Comment:
We shouldn't use the names of objects in Perun for the identification of objects in the external system. For this purpose, you can use a new attribute, for example 'resource:def:atlassianGroupName' set on the Resource where groups are assigned.

Reply:
fixed and simplified the script to skip resolving each group separately and rather process whole resource instead